### PR TITLE
Implemented the balck stripe title on all pages

### PR DESCRIPTION
**Pull request by baxgas**

### CODE REVIEW

**Comment by gammaw** [comment]
@baxgas looks good! Can you resolve the conflicts before I approve? Otherwise we'll need a new approval I guess.

**Comment by baxgas** [comment]
@gammaw yes, please re-approve, to be able to merge

**Review by gammaw — APPROVED** [review]
Thanks!

**Pull request merged**
